driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -234,6 +241,23 @@ private void setFinalResult(
     }
   }
 
+  private void logServerWarnings(List<String> warnings) {
+    // log the warnings if they have NOT been disabled (double negative)
+    if (!Boolean.getBoolean(RequestHandler.DISABLE_QUERY_WARNING_LOGS)) {
+      // comma separated list of warnings
+      StringBuilder warningString = new StringBuilder();
```

Comment:
Use Guava's `Joiner` instead.

Reply:
I decided to log each warning in the list separately, so there is no need for joining anymore.